src/main/java/org/elasql/perf/tpart/metric/MetricCollector.java

```diff
@@ -53,8 +59,11 @@ public void run() {
 	private TPartSystemMetrics collectSystemMetrics() {
 		TPartSystemMetrics.Builder builder = new TPartSystemMetrics.Builder(Elasql.serverId());
 
-		// XXX: for demo
-		builder.setFakeMetric(12345);
+		builder.setProcessCpuLoad(bean.getProcessCpuLoad());
+		builder.setSystemCpuLoad(bean.getSystemCpuLoad());
+		builder.setSystemLoadAverage(bean.getSystemLoadAverage());
+		
+		builder.setThreadActiveCount(Thread.activeCount());
```

Comment:
The number outputs from here seems weird in my experiment. I used only 1 RTE and 1 client process, but the report shows that there were 1010 threads in the end of the experiment. Cloud you check if this method also counts the **waiting** threads?

Reply:
What is the thread pool size of the experiment.
I'll check it.

Reply:
In my experiment: 2 RTE , 2 client Process and 2 servers, `transaction-features.csv` shows that the active thread count is `64` which seems to be normal.

Reply:
Weird, but `64` is still way too many for 2 RTEs and 2 clients. How long did the experiment run?